### ISSUE TITLE
Problem with image include during site build

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/react-static-plugin-aperis-doc-pages#7]
From @w00lf:

Example of builds: https://github.com/lutaml/lutaml.org/actions/runs/480515518, https://github.com/lutaml/lutaml.org/actions/runs/480515516. This is the page that has an image inserted in it, but during build process the following error is thrown and the build is halted:

```
[✓] HTML Exported (4.5s)
Copying media docs/uml-domain-specific-languagesimple-tutorial/tutorial_diagram_1.png dist//uml-domain-specific-languagesimple-tutorial/tutorial_diagram_1.png
  Error: Trace: Error: ENOENT: no such file or directory, copyfile 'docs/uml-dom  ain-specific-languagesimple-tutorial/tutorial_diagram_1.png' -> 'dist//uml-dom  ain-specific-languagesimple-tutorial/tutorial_diagram_1.png'
      at Object.copyFileSync (fs.js:1991:3)
      at _callee2$ (/home/runner/work/lutaml.org/lutaml.org/lutaml-dot-org/node_  modules/@riboseinc/react-static-plugin-aperis-doc-pages/node.api.js:195:44)
      at tryCatch (/home/runner/work/lutaml.org/lutaml.org/lutaml-dot-org/node_m  odules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (/home/runner/work/lutaml.org/lutaml.org/  lutaml-dot-org/node_modules/regenerator-runtime/runtime.js:293:22)
```

Originally at https://github.com/paneron/paneron/issues/19

**Comment by strogonoff** [comment]
That site [used an old version of react-static plugin](https://github.com/lutaml/lutaml.org/blob/24f6bfe01500f97eef1b5a20f19ba4dd21c38894/lutaml-dot-org/package.json#L16), updated to 1.0.23 in https://github.com/lutaml/lutaml.org/commit/2c2312d69940cb77b19ed9b4c20dfc38f5a5de41.

**Comment by strogonoff** [comment]
(Fixed in 1.0.24 and updated LutaML site’s package.json)
